components/page-aside/PageAside.vue

```diff
@@ -37,11 +45,25 @@
 <script>
 import _defaultAvatarSrc from './assets/Portrait_Placeholder.png'
 
+const sizeLimit = 4 * 20
+const imgScale = function (imgObj) {
```

Comment:
you could make this a method so you don't need the computed, also I'd rather see the actual properties used, either via destructuring or via hard parameters.